tests/unit/controlgroup/controlgroup_methods.js

```diff
@@ -0,0 +1,120 @@
+module( "Controlgroup: methods" );
+
+test( "destroy", function() {
+	expect( 1 );
+	domEqual( ".controlgroup", function() {
+		$( ".controlgroup" ).controlgroup().controlgroup( "destroy" );
+	});
+});
+test( "disable", function( assert ) {
+	expect( 2 );
+	var element = $( ".controlgroup" ).controlgroup().controlgroup( "disable" );
+	assert.lacksClasses( element, "ui-state-disabled",
+		"ui-state-disabled is not present on widget after disabling" );
+	strictEqual( element.find( ".ui-state-disabled" ).length, 6,
+		"Child widgets are disabled" );
+});
+
+test( "enable", function( assert ) {
+	expect( 2 );
+	var element = $( ".controlgroup" ).controlgroup().controlgroup( "enable" );
+	assert.lacksClasses( element, "ui-state-disabled",
+		"ui-state-disabled is not present on widget after enabling" );
+	strictEqual( element.find( "ui-state-disabled" ).length, 0,
+		"Child widgets are disabled" );
+});
+
+function hasCornerClass( element, className ) {
+	if ( className ) {
+		return element.hasClass( className );
+	}
+
+	return !!element.attr( "class" ).match( /ui-corner/g );
+}
+test( "refresh", function() {
```

Comment:
Let me know when you've dealt with some/most of the other issues, that might provide some inspiration for this test.